src/seedu/addressbook/AddressBook.java

```diff
@@ -1163,5 +1193,22 @@ private static String removePrefixSign(String s, String sign) {
     private static ArrayList<String> splitByWhitespace(String toSplit) {
         return new ArrayList<>(Arrays.asList(toSplit.trim().split("\\s+")));
     }
+    /*
+     * Capitalizes first letter of the string
+     */
+    private String capitalizeFirstLetter(String stringToBeCapitalized) {
+	return Character.toUpperCase(stringToBeCapitalized.charAt(0)) + stringToBeCapitalized.substring(1);
+    }
+    
+    /*
+     * Capitalizes each string in a given arraylist of String
+     */
+    private ArrayList<String> capitalizeEachWord(ArrayList<String> stringsToBeCapitalized) {
+	ArrayList<String> capitalizedString = new ArrayList<String>();
+	for(String i : stringsToBeCapitalized) {
```

Comment:
colon here should have space on the right only, in your case:

    for (String word: wordsToBeCapitalized) {